product/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls   import path
+from product.views import (
+    ProductDetailView, 
+    ProductListView, 
+    SellerItemsView, 
+    ProductUploadView
+)
+from django.conf.urls.static import static
+from django.conf import settings
+
+urlpatterns = [path('/detail', ProductDetailView.as_view()),
```

Comment:
```suggestion
urlpatterns = [path('/<int:product_id>', ProductDetailView.as_view()),
```
세부 정보에 접근할 때는 `product/` 뒤에 제품의 번호가 오는 것이 RESTful API 조건에 적합합니다 :)

Reply:
반영완료!